Makefile

```diff
@@ -13,25 +20,29 @@ openwrt/.config: openwrt/feeds.conf
 	cat creator-kit.config > ../dist/openwrt/.config
 	$(MAKE) -C ../dist/openwrt defconfig
 
-.PHONY: openwrt
-openwrt: openwrt/.config
+.PHONY: build_openwrt
+build_openwrt: openwrt/.config
 	if test $(findstring J=,$(MAKEFLAGS)); then \
 		$(MAKE) $(SUBMAKEFLAGS) -C ../dist/openwrt -j$(J); \
 	else \
 		$(MAKE) $(SUBMAKEFLAGS) -C ../dist/openwrt; \
 	fi;
 
 # Building Contiki apps
-.PHONY: contiki
-contiki:
+.PHONY: build_contiki
+build_contiki:
 	$(MAKE) -C ../packages/led-actuator TARGET=mikro-e
 	$(MAKE) -C ../packages/button-sensor TARGET=mikro-e
 
-.PHONY: copy_binaries
-copy_binaries:
+# Copy files to build/output/
+.PHONY: copy_contiki
+copy_contiki: build_contiki
 	mkdir -p output/contiki
 	cp ../packages/led-actuator/lwm2m-client-led-actuator.hex output/contiki/
 	cp ../packages/button-sensor/lwm2m-client-button-sensor.hex output/contiki/
+
+.PHONY: copy_openwrt
+copy_openwrt: build_openwrt
 	mkdir -p output/openwrt
 	find ../dist/openwrt/bin/pistachio/ -maxdepth 1 -type f -exec cp {} output/openwrt/ \;
```

Comment:
Please add one more line above 'find' step 
- zip -r output/openwrt/packages.zip ../dist/openwrt/bin/pistachio/packages/*

This is required to provide IPK packages in a zip for uploading them to the OPKG server